### DISCUSSION TITLE
Import of Favourites from old Maps.me

### DISCUSSION THREAD

**Discussion by komape** [organicmaps/organicmaps#638] [Q&A]
I just found this Maps.me alternative. Thanks formthe work 🙏 I really would like to switch to this and get rid of the old Maps.me. But one important thing for me are the favourite places. I have around 1000 of them, so it's not an option to transfer them manually. Instead it's possible to export them as a '.kmz' file. Unfortunately I couldn't find an option to import that file. 

How about making it possible to import that file into Organic Maps?

So far the favourites are more important to me and I'll continue with Maps.me. This feature would make me switch.

**Comment by matrixik** [comment]
I just did exactly this. You have "Import bookmarks" under "Create new list":
![Screenshot_20210618-152647.png](https://user-images.githubusercontent.com/201930/122568803-6aa09c00-d04a-11eb-9c8a-d96c8dad3eea.png)

**Comment by ted-piotrowski** [comment]
Import Bookmarks does not exist on iOS app. At least the one published to the App Store. @matrixik are you using a Play Store build or a nightly build?

**Comment by ted-piotrowski** [comment]
@komape I successfully exported on iOS. Here's the procedure:

```
1. Click on ... menu next to My Places in Maps.me and choose export file
2. In the Share menu that pops up choose to share to Organic Maps app
```

**Comment by rtsisyk** [comment]
>Import Bookmarks does not exist on iOS app. 

Import Bookmarks only exists on Android. On iOS there is no way to access other' app file system. 

We prepare an instruction:

![mm-migrate-ios](https://user-images.githubusercontent.com/1799054/122632116-3f18c280-d0d9-11eb-81e6-54ffcab34e97.jpg)

**Comment by rtsisyk** [comment]
I moved this ticket to Q&A to help other people to find the answer.

**Comment by rtsisyk** [comment]
@komape could you please mark this question as answered?
